app/arguments.py

```diff
@@ -1,113 +1,95 @@
 #!/usr/bin/env python3
-"""Allows to pass arguments to the drucker command"""
+"""Parsing of command line arguments passed to the drucker."""
 
-import subprocess as s
+import sys
 import argparse
-import colorful as c
-import variables as v
-import orchestration as o
+import subprocess
```

Comment:
`subprocess as sub`

Reply:
I recommend to not do this, while its totally common to alias your own modules, it becomes less conventient for third-party developers to not be able to recognize libraries like `subprocess` and `colorful`. Its better to only do this for your own modules, but obviously your choice :-)